### ISSUE TITLE
Command not found on Ubuntu

### ISSUE THREAD

**Issue by lhrossi** [sheerun/bower-away#31]
I followed the usage instructions and installed bower-away using yarn and then using then `npm install -g bower-away`, but when I type bower-away I get this error.

Should I modify something in my environment variables?

**Comment by lhrossi** [comment]
Got it, my node libs was not on the PATH variable: `export PATH="$PATH:/home/XXXXXX/public/node_modules/bin"`

**Comment by ay11111111111** [comment]
Hi, could you, please, provide more details how did you solve the problem? I have the same issue. @lhrossi

**Comment by lhrossi** [comment]
Yes, you need to put your node libraries accessible to the application, what I did was to put it on the environment variable PATH, just like I put above.
